mk/spksrc.archs.mk

```diff
@@ -15,7 +17,7 @@ GENERIC_x64_ARCH = x64
 GENERIC_ARCHS = $(GENERIC_ARMv7_ARCH) $(GENERIC_ARMv8_ARCH) $(GENERIC_x64_ARCH)
 
 ARMv5_ARCHS = 88f6281
-ARMv7_ARCHS = $(GENERIC_ARMv7_ARCH) $(DSM_ARMv7_ARCHS) $(SRM_ARMv7_ARCHS)
+ARMv7_ARCHS = $(GENERIC_ARMv7_ARCH) $(DSM_ARMv7_ARCHS) $(SRM_ARMv7_ARCHS) $(OLD_DSM_ARMv7_ARCHS)
```

Comment:
Correct me if I'm wrong but if comcerto2k (`$(OLD_DSM_ARMv7_ARCHS)`) is included here it is still part of a generic arch.